lib/countly.js

```diff
@@ -1595,16 +1597,17 @@
 
     //insert request to queue
     function toRequestQueue(request) {
-        //ignore bots
-        if (Countly.ignore_visitor) {
+        
+        // disable this controls if test_mode is active
+        if (!Countly.test_mode && Countly.ignore_visitor) {
```

Comment:
I don't think we need this one, as it would prevent data from entering `requestQueue` at all, no?